src/utils/interfaces/Place.ts

```diff
@@ -35,6 +35,9 @@ interface Experience {
   id: number;
   placeId: number;
   userId: number;
+  username: string;
+  dateVisited: Date;
+  etiquette: string;
```

Comment:
Can likely change this later, but might this use the etiquette ID as opposed to the label/string?

Reply:
agreed